swagger_ui_bundle/vendor/swagger-ui-3.52.0/index.html

```diff
@@ -33,13 +33,13 @@
   <body>
     <div id="swagger-ui"></div>
 
-    <script src="./swagger-ui-bundle.js"> </script>
-    <script src="./swagger-ui-standalone-preset.js"> </script>
+    <script src="./swagger-ui-bundle.js" charset="UTF-8"> </script>
+    <script src="./swagger-ui-standalone-preset.js" charset="UTF-8"> </script>
     <script>
     window.onload = function() {
       // Begin Swagger UI call region
       const ui = SwaggerUIBundle({
-        url: "https://petstore.swagger.io/v2/swagger.json",
+        url: "http://localhost:5000/openapi.json",
```

Comment:
@aparcar, I think it better to revert the standard URL in this place?
`https://petstore.swagger.io/v2/swagger.json` is more general for all users of `swagger_ui_bundle`.

Reply:
Updated.